README.md

```diff
@@ -163,6 +163,7 @@ Nearly always hiring in Newcastle & Edinburgh (UK).
 * [Day One](https://www.dayoneapp.com/)
   ([GitHub](https://github.com/bloom)) -
   Day One is a simple and beautiful way to archive your life.
+* [Level](https://level.app) ([GitHub](https://github.com/levelhq/level)) - Distraction-free communication for software teams.
```

Comment:
Could you wrap this line at 80 columns please? :)

Reply:
Fixed